docs/variables/downtime.md

```diff
@@ -0,0 +1,39 @@
+---
+id: downtime
+---
+
+# $(downtime)
+
+Returns the time that has passed since a user's stream has gone offline.
+
+#### Parameters
+
+This variable takes **one** *optional* parameter that is a Twitch username of who to fetch the downtime of. Defaults to the sender's username if not provided.
```

Comment:
It doesn't take optional parameters. our uptime/downtime isn't powered by twitch, we use our own cache that we collect from webhooks, so we can't check other users.

Reply:
Makes sense, I'll remove the parts indicating that it's possible.

Reply:
was this done